modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -375,6 +362,7 @@ function transformVertexNormals(vertexVector, calleeArgs) {
 function flattenTexCoords(texCoords, indices) {
   const newTexCoords = new Float32Array(indices.length * VALUES_PER_TEX_COORD);
   if (!texCoords) {
+    newTexCoords.fill(1);
```

Comment:
A bit strange to have uv coordinates if there are no textures... but ok. 

Explain in comment that we are returning dummy UVs?

Reply:
Done